### DISCUSSION TITLE
dynamic grids with bootstrap

### DISCUSSION THREAD

**Discussion by d3vsuthar** [bootstrap-email/bootstrap-email#129] [Help]
I plan to use bootstrap email along with liquid tags to loop through a bunch of images
My bootstrap email code is set up this way

- container
	- row
		- more than 3 divs with class *col-lg-3*

This works fine on mobile screens (with the lg cols), however on desktop - I can only see the first 3 divs
Is there a way I can get this to show up like this

```
div1 div2 div3
div4 div5 div6
```

**Comment by stuyam** [comment]
I would suggest making a separate html row for each row on desktop. So if you are doing 3 columns on desktop I would loop through in groups of 3 and make a row for each. Idk off the top of my head what the liquid code would look like but I assume you could use something like using modulo 3 `% 3` or something and chunk the images into groups of 3 dynamically.

Also if you are wanting 3 columns you want to use `col-lg-4` because it is out of a 12 column grid.
